### PR TITLE
Update the doc to introduce cloud value

### PR DESCRIPTION
**Pull request by jiaochenlu**
Update the doc to introduce cloud value

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by jiaochenlu** [comment]
@wangchao1230 and @henry-zeng 
pls help to review the doc update, to introduce the value of cloud version, when and how to transition the flow from local to cloud

**Review by ChenJieting — APPROVED** [review]

File: .cspell.json [hunk 1]
```diff
@@ -145,7 +145,8 @@
     "Hwfoxydrg",
     "llms",
     "vcrpy",
-    "uionly"
+    "uionly",
```
Comment (crazygao) [inline .cspell.json:148]: this seems not related

Comment (jiaochenlu) [reply .cspell.json]: removed

**Review by wangchao1230 — APPROVED** [review]

**Review by ChenJieting — APPROVED** [review]
